sdk/src/main/java/io/horizen/account/api/rpc/types/FeePaymentsView.java

```diff
@@ -15,17 +16,30 @@ public FeePaymentsView(AccountFeePaymentsInfo info) {
         payments = JavaConverters
             .seqAsJavaList(info.payments())
             .stream()
-            .map(payment -> new FeePaymentData(payment.address().address(), payment.value()))
+            .map(payment -> FeePaymentData.fromAccountFeePayment(payment))
             .collect(Collectors.toList());
     }
 
     private static class FeePaymentData {
         public final Address address;
         public final BigInteger value;
+        public final BigInteger valueFromMainchain;
+        public final BigInteger valueFromFees;
 
-        public FeePaymentData(Address address, BigInteger value) {
+        public FeePaymentData(Address address, BigInteger value, BigInteger valueFromMainchain, BigInteger valueFromFees) {
             this.address = address;
             this.value = value;
+            this.valueFromMainchain = valueFromMainchain;
+            this.valueFromFees = valueFromFees;
+        }
+
+        public static FeePaymentData fromAccountFeePayment(AccountPayment payment) {
+            return new FeePaymentData(
+                payment.address().address(),
+                payment.value(),
+                payment.valueFromMainchain().getOrElse(() -> BigInteger.ZERO),
```

Comment:
Is it possible to filter out in the json response `valueFromMainchain` and  `valueFromFees` when they are None? Because putting 0 seems misleading in this case. Or maybe returning -1 instead of 0, just to make more explicit that these 2 fields are not meaningful before the fork.

Reply:
Found the Serializer, will try to hide the field completely. If not possible, will change to -1 for missing value

Reply:
Added a custom serizlized module for java optionals, now empty will be filtered. Thanks!